packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -4,7 +4,7 @@ title: Get a refresh token
 
 To get a refresh token, you send a request to your Okta Authorization Server.
 
-The only flows that support refresh tokens are the resource owner password flow and the authorization code flow. This means that the following combinations of grant type and scope, when sent to the `/token` endpoint, return a refresh token:
+The only flows that support refresh tokens are the authorization code flow or the resource owner password flow. This means that the following combinations of grant type and scope, when sent to the `/token` endpoint, return a refresh token:
```

Comment:
Would it make more sense to say `authorization code flow and the resource owner password`? It sounds like you are enumerating what flows that supports refresh tokens, so using `or` sounds odd to me